From c5f8b56d2820a6733e1bde6637912dd7b820ea80 Mon Sep 17 00:00:00 2001
From: Patrick McCarty <patrick.mccarty@linux.intel.com>
Date: Mon, 9 Dec 2013 17:09:27 -0800
Subject: [PATCH 10/24] user@.service: use the SmackExecLabel key

On Tizen, all user@.service instances should run with the User label, so
use the new SmackExecLabel key to set it at runtime.

Note: This patch should remain downstream, since it relies on the
Tizen-specific three-domain model.

Change-Id: I59b9bfa3f7542c8eed6f768fe0fba153ad494747
Signed-off-by: Patrick McCarty <patrick.mccarty@linux.intel.com>
---
 units/user@.service.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/units/user@.service.in b/units/user@.service.in
index 8091ce1..08b1cd7 100644
--- a/units/user@.service.in
+++ b/units/user@.service.in
@@ -14,5 +14,6 @@ User=%i
 PAMName=systemd-user
 Type=notify
 ExecStart=-@rootlibexecdir@/systemd --user
+SmackExecLabel=User
 Slice=user-%i.slice
 KillMode=mixed
-- 
1.8.4.5

